packages/ui/src/components/DateInput/__stories__/Range.stories.tsx

```diff
@@ -0,0 +1,17 @@
+import { Template } from './Template'
+
+export const Range = Template.bind({})
+
+Range.parameters = {
+  docs: {
+    storyDescription:
+      'With `startDate` and `endDate` you can define a range of dates',
+  },
+}
```

Comment:
This is the old way to describe a story please use this instead:

```suggestion
Range.parameters = {
  docs: {
    description: {
          story: 'With `startDate` and `endDate` you can define a range of dates',
      }
  },
}
```